base/Comfy.js

```diff
@@ -0,0 +1,333 @@
+const { Client, Collection, MessageEmbed } = require('discord.js');
+const { GiveawaysManager } = require('discord-giveaways');
+const { Player } = require('discord-player');
+const NodeCache = require("node-cache");
+const factorioServers = require("../servers");
+const FIFO = require("fifo")
+
+const util = require('util'),
+  path = require('path'),
+  AmeClient = require("amethyste-api"),
+  moment = require('moment');
+
+class Comfy extends Client {
+  constructor(options) {
+    super(options)
+    this.consts = require("../consts")
+    this.config = require('../config');
+    this.emotes = this.config.emojis;
+
+    this.commands = new Collection();
+    this.aliases = new Collection();
+
+    this.logger = require('../helpers/logger');
+    this.functions = require('../helpers/functions');
+    this.wait = util.promisify(setTimeout);
+
+    this.guildsData = require('./Guild');
+    this.usersData = require('./User');
+    this.membersData = require('./Member');
+    this.logs = require('./Log');
+
+    this.queues = new Collection();
+
+    this.serverQueues = new Map();
+    factorioServers.forEach((server) => {
+      if (!server.discordid) return
+      this.serverQueues.set(server.discordid, {
+        server: server,
+        messageQueue: FIFO(),
+        sendingMessage: false
+      })
+    })
+    setInterval(() => {
+      this.serverQueues.forEach((server) => {
+        if (server.sendingMessage === true) return
+        server.sendingMessage = true
+        let message = ""
+        while (!server.messageQueue.isEmpty()) {
+          let content = server.messageQueue.first()
```

Comment:
`content` is only used once. Perhaps just replacing content with what you have assigned to it would would read easier